### ISSUE TITLE
Tacotron-2 implementation with Tensorflow 2 and Tensorflow Addons

### ISSUE THREAD

**Issue by dathudeptrai** [tensorflow/addons#1906]
**Describe the feature and the current behavior/state.**
Currently, i see no good example use Tensorflow addons seq2seq on Speech domain (Like Text To Speech problem). TTS is a big problem on speech and many people are interested. I just released TensorflowTTS: Real-Time State-of-the-art Speech Synthesis for Tensorflow 2. I want my use my implementation to prove the effectiveness of tf_addons seq2seq and Tensorflow 2. But i don't know how to add my implementation to the tutorial :D.

**Relevant information**
- Are you willing to contribute it (yes/no): yes
- Are you willing to maintain it going forward? (yes/no): yes
- Is there a relevant academic paper? (if so, where): https://arxiv.org/abs/1712.05884
- Is there already an implementation in another framework? (if so, where): https://github.com/mozilla/TTS, https://github.com/NVIDIA/tacotron2/
- Was it part of tf.contrib? (if so, where): no

**Which API type would this fall under (layer, metric, optimizer, etc.)**
tfa seq2seq, layer, optimizer.
**Who will benefit with this feature?**
Researcher, people care about Text to speech problem and want to deploy TTS model.
**Any other info.**
https://github.com/dathudeptrai/TensorflowTTS

**Comment by seanpmorgan** [comment]
Hi @dathudeptrai! This is a very cool!  Are you requesting to [add a new tutorial](https://github.com/tensorflow/addons/tree/master/docs/tutorials) to the repository? If so that would be much appreciated, but we don't want to add another repository as a dependency in the tutorials for our repo. Is it possible to make a minimal example in a Colab notebook without external deps? If so it'd be fine to add a note in the tutorial pointing to the TTS library you've created but not a requirement

**Comment by dathudeptrai** [comment]
hi @seanpmorgan, i can create Colab notebook for define model, inference part (load pretrained and do inference, visualize Attention map.). It's not ez to create training process on colab, there are many related code like TF data, preprocessing .... So is that ok if i create colab for modeling and inference part, for training it's fine to add a note in tutorial pointing to the TTS library ?

**Comment by seanpmorgan** [comment]
Hi @dathudeptrai unfortunately it's difficult to use pre-trained weights in colab in a way that allows everyone to run the model (would have to be hosted somewhere). Just from that description of the notebook I'm leaning toward it not being a great fit for our tutorials, but if you want to make a notebook that you think is still useful I'd be happy to take a look.

Don't want to waste your time though because we're generally looking for end to end tutorials for users. Perhaps an external contribution to TF Hub would be a good place for the model? If it were accepted then we could certainly add a tutorial that used it.

**Comment by dathudeptrai** [comment]
ok i will consider tf hub
